### DISCUSSION TITLE
IntelliJ Visual editor plugin

### DISCUSSION THREAD

**Discussion by lengoman** [statelyai/xstate#4821] [Ideas]
It would be cool to have the awesome VS Code Xstate plugin available for IntelliJ

**Comment by davidkpiano** [comment]
I agree. We're currently working on updating the current extension and thinking about ways to make it work for other IDEs.

**Comment by dsmurl** [comment]
Many at my company use webstorm adn jetbrains, a jetbrains pluugin would help adoption of xstate.  Any progress or timeline on this?
